optimum/utils/input_generators.py

```diff
@@ -296,6 +296,7 @@ class DummySeq2SeqDecoderTextInputGenerator(DummyDecoderTextInputGenerator):
         "decoder_input_ids",
         "decoder_attention_mask",
         "encoder_outputs",
+        "attention_mask",
```

Comment:
Can't we use the regular `DummyEncoderTextInputGenerator` in places where we need that instead?

Reply:
Yes, not sure why I removed the encoder text generator. I have updated the input generator.